packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -44,15 +44,15 @@ export function getBlockMoverDescription( selectedCount, type, firstIndex, isFir
 
 	if ( isFirst && isLast ) {
 		// translators: %s: Type of block (i.e. Text, Image etc)
-		return sprintf( __( 'Block %s is the only block, and cannot be moved' ), type );
+		return sprintf( __( 'Block %s is the only block, and cannot be moved' ), blockLabel );
 	}
 
 	if ( dir > 0 && ! isLast ) {
 		// moving down
 		return sprintf(
 			// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: Direction of movement ( up, down, left, right ), 4: New position
 			__( 'Move %1$s block from position %2$d %3$s to position %4$d' ),
```

Comment:
The block mover screen reader text ('Move block from position 1 to position 2') presents a bit of a conundrum. There's a desire to use the accessible block label here because it incorporates info about the content of the block. For example if you had three image blocks in a post, the user needs to know which one they're moving. 

However, for some of the blocks like paragraph the accessibility label is very long, and all the pertinent information about where the block is moved to or from is at the end of this text. A user won't want to hear an entire paragraph before hearing where the block will be moved to.

Some options:
1. Reformat this copy so the block label is at the end, e.g. 'Move from position 1 to 2, Paragraph Block. Lorum Ipsum ...'. It doesn't sound quite as good!
2. Offer a way to truncate the accessible block label.

I think the second option would be preferable.

Reply:
I think for the movers we could leave out the accessible description altogether, and just have the name of the block. I've been playing around with VoiceOver for a bit and AFAICT, to get to the movers, you always have to focus on the block itself first, so you'll already know from the block aria-label which block you're on. 

Also, the word "block" is already part of the block names, so perhaps we could edit the string to be "Move %1$s from position..." to avoid repetition.

Reply:
I've switched it back to using the block title, so it'll no longer have the repetition issue.